packages/clay-css/src/scss/cadmin/variables/_buttons.scss

```diff
@@ -701,6 +701,19 @@ $cadmin-btn-palette: map-deep-merge(
 		light: $cadmin-btn-light,
 		dark: $cadmin-btn-dark,
 		link: $cadmin-btn-link,
+		btn-beta: (
+			background-color: $cadmin-gray-100,
```

Comment:
Hey @pat270, design has requested that the background colors for these actually use opacity to define the color so that it properly renders in a variety of different contexts where the background color could be light or dark. IE: $info-d2 4%

Reply:
@ethib137 I tried the opacity, but it fails under dark backgrounds.

Reply:
Updated the colors!